### PR TITLE
Added the timeout function to all pages in the calculation.

### PR DESCRIPTION
**Pull request by DavidWoodward37**

### CODE REVIEW

**Comment by MJCallahanPage** [comment]
Hello @DavidWoodward37 can you resolve the merge conflicts please. :banana:

**Comment by MJCallahanPage** [comment]
Now waiting for Acceptance Test changes

File: test/controllers/TimeoutControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+class TimeoutControllerSpec extends UnitSpec with WithFakeApplication {
+
+  class fakeRequestTo(url : String, controllerAction : Action[AnyContent]) {
+    val fakeRequest = FakeRequest("GET", "/calculate-your-capital-gains/" + url)
+    val result = controllerAction(fakeRequest)
+    val jsoupDoc = Jsoup.parse(bodyOf(result))
+  }
+
+//  class fakeRequestToWithSessionId(url : String, controllerAction : Action[AnyContent]) {
```
Comment (macgrewal) [inline test/controllers/TimeoutControllerSpec.scala]: Remove comments

File: test/controllers/CalculationControllerTests/CurrentIncomeSpec.scala [hunk 2]
```diff
@@ -141,6 +141,19 @@ class CurrentIncomeSpec extends UnitSpec with WithFakeApplication with MockitoSu
     }
   }
 
+  "In CalculationController calling the .currentIncome action " when {
```
Comment (macgrewal) [inline test/controllers/CalculationControllerTests/CurrentIncomeSpec.scala]: encapsulate assertion  in should so that it reads better

**Comment by MJCallahanPage** [comment]
Do we really need to use the ValidSession check on the post requests? If they have gotten to the page then surely they will have an active session? Or does this cater for a user who leaves their browser open for a long period of time so that the session has expired?

**Pull request merged**